packages/flame/lib/src/components/scroll_text_box_component.dart

```diff
@@ -0,0 +1,157 @@
+import 'dart:async';
+
+import 'package:flame/components.dart';
+import 'package:flame/events.dart';
+import 'package:flame/text.dart';
+import 'package:flutter/painting.dart';
+
+/// [ScrollTextBoxComponent] configures the layout and interactivity of a
+/// scrollable text box.
+/// It focuses on the box's size, scrolling mechanics, padding, and alignment,
+/// contrasting with [TextRenderer],
+/// which handles text appearance like font and color.
+/// This component uses [TextBoxComponent] to provide
+/// scrollable text capabilities.
```

Comment:
Line break when the word would go past 80, not before :)

Reply:
This one wasn't fixed, the lines starting with `which` and `scrollable` should continue on the previous lines, until they hit 80